src/TurbulenceClosures/turbulence_closure_implementations/anisotropic_minimum_dissipation.jl

```diff
@@ -63,7 +63,7 @@ Keyword arguments
         buoyancy modification term. This implementation differs from [Abkar16](@citet)'s proposal
         and the impact of this approximation has not been tested or validated.
 
-By default: `C = Cν = Cκ = 1/12`, which is appropriate for a finite-volume method employing a
+By default: `C = Cν = Cκ = 1/3`, which is appropriate for a finite-volume method employing a
 second-order advection scheme, and `Cb = nothing`, which turns off the buoyancy modification term.
```

Comment:
@jagoosw please add text here from the issue, with appropriate citations, that explain why 1/3 is the right choice

Reply:
Also put in a reference to https://github.com/CliMA/Oceananigans.jl/issues/4367 which has some relevant plots

Reply:
The discussion will be an elaboration of this remark:

"I think they're getting there a different way but that in equation 39 they're finding the equivalent of setting it to 1/3?"